app/javascript/components/SearchPanel.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import KeyWordSearch from './KeyWordSearch';
+import Button from 'react-bootstrap/lib/Button';
+// import FacetControl from './FacetControl';
+// import MoreFiltersButton from './MoreFiltersButton';
+import Grid from 'react-bootstrap/lib/Grid';
+import Row from 'react-bootstrap/lib/Row';
+import Col from 'react-bootstrap/lib/Col';
+import { faNewspaper, faChevronLeft } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+// Only for development!  We'll fetch data once API endpoints are available.
+import {facetsResponseMock, searchFiltersResponseMock} from './FacetsMockData';
+const facets = facetsResponseMock;
+
+const defaultFacetIDs = ['disease', 'organ', 'species', 'cell_type'];
+const moreFacetIDs = ['sex', 'race', 'library_preparation_protocol', 'organism_age'];
+
+const defaultFacets = facets.filter(facet => defaultFacetIDs.includes(facet.id));
+const moreFacets = facets.filter(facet => moreFacetIDs.includes(facet.id));
+
+window.searchFiltersResponse = searchFiltersResponseMock;
+
```

Comment:
I'd rather not have css declared in javascript components except as a last resort